lib/offline.js

```diff
@@ -42,7 +45,7 @@ module.exports = promisify(function(config, callback) {
   // Remove the existing service worker, if any, so sw-precache doesn't include
```

Comment:
Nit: update comments that reference *sw-precache*, like this one, to reference *offliner* instead.